README.md

```diff
@@ -32,6 +36,18 @@ $ npm install -g promise-it-wont-hurt@latest
 **Note**: the workshop is being updated weekly. 
 Please regularly rerun the above command to get the latest set of exercises.
 
+### 2. Docker
+
+```
+alias workshop='docker run -it -v $(pwd):/root oreng/promises-workshop'
```

Comment:
You know what would be good, we can check in the Dockerfiles directly into this repo, so we can maintain it if say you are not reachable. I'm not familiar with Docker, but do you think that is doable?